app/controllers/account/billing-info/invoices/list.js

```diff
@@ -0,0 +1,161 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
+
+export default class extends Controller.extend(EmberTableControllerMixin) {
```

Comment:
Check alignment on this file. Seems misaligned.
Also, use the model directly instead of an unnecessary computed property in between